### ISSUE TITLE
"You do not have the required payment"

### ISSUE THREAD

**Issue by Zillew** [phaed420/PreciousStones#803]
I have a feeling it has to do with vault (we were on 1.5.2, but now 1.5.3).

Basically when trying to rent, buy, or use a shared sign, the message "You do not have the required payment" shows. We just update to #487, and were previously on #422 (it worked on 422).

Nothing in console.

Thanks!

**Comment by phaed420** [comment]
What version of vault are you running?

**Comment by Zillew** [comment]
1.5.3, and 1.5.2 didn't work either.

**Comment by phaed420** [comment]
Try the new build coming off of Jenkins.

**Comment by Zillew** [comment]
Still not working :/

**Comment by phaed420** [comment]
Maybe you need to upgrade your economy plugin?

**Comment by Zillew** [comment]
We are using Essentials Eco, and are on the latest build on their Jenkins page... The bridge between our other plugins (Such as QuickShop) which rely on Vault & our eco plugin are still working without problems :S

**Comment by Zillew** [comment]
Updated to new PreciousStones jenkins build.
Tried with 1.5.3 Vault - Same problem
Tried with 1.5.2 Vault - Same problem
Tried with 1.4.1 Vault - Same problem

Doesn't seem to be fixed... :/

**Comment by Zillew** [comment]
To rule out the possibility of a problem with all transaction through the plugin, I tested using an item payment instead. 

Item payments seem indeed to be working fine.

**Comment by Zillew** [comment]
I believe I think I may know where the problem is. 
Before I updated, I was having issues with item transactions (using sell/rent/share signs). Basically the transaction wasn't physically taking anything, just saying it was. 
This has now been fixed in the later versions. What I'm thinking is that when we now type in an economy price, it's looking for an item ID with that value, or something like that.
